.travis.yml

```diff
@@ -5,31 +5,40 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
 env:
-  - SYMFONY_VERSION=2.3.*
+  - SYMFONY_VERSION=2.7.*
 
 matrix:
-  allow_failures:
-    - env: SYMFONY_VERSION=dev-master
   include:
-    - php: 5.5
+    - php: 5.6
       env: SYMFONY_VERSION=2.3.*
-    - php: 5.5
+    - php: 5.6
       env: SYMFONY_VERSION=2.4.*
-    - php: 5.5
+    - php: 5.6
       env: SYMFONY_VERSION=2.5.*
```

Comment:
you should remove testing against 2.4, 2.5 and 2.6 which are EOLed IMO, to limit the number of jobs in the matrix